File: app/src/main/java/vn/loitp/app/activity/customviews/ariana/ArianaMenuActivity.java
package vn.loitp.app.activity.customviews.ariana;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import loitp.basemaster.R;
import vn.loitp.app.activity.customviews.ariana.iv.ArianaImageViewActivity;
import vn.loitp.app.activity.customviews.ariana.tv.ArianaTextViewActivity;
import vn.loitp.app.activity.customviews.ariana.vp.ArianaViewPagerActivity;
import vn.loitp.core.base.BaseActivity;
import vn.loitp.core.utilities.LActivityUtil;

public class ArianaMenuActivity extends BaseActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        findViewById(R.id.bt_ariana_imageview).setOnClickListener(this);
        findViewById(R.id.bt_ariana_textview).setOnClickListener(this);
        findViewById(R.id.bt_ariana_viewpager).setOnClickListener(this);
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_menu_ariana;
    }

    @Override
    public void onClick(View v) {
        Intent intent = null;
        switch (v.getId()) {
            case R.id.bt_ariana_imageview:
                intent = new Intent(activity, ArianaImageViewActivity.class);
                break;
            case R.id.bt_ariana_textview:
                intent = new Intent(activity, ArianaTextViewActivity.class);
                break;
            case R.id.bt_ariana_viewpager:
                intent = new Intent(activity, ArianaViewPagerActivity.class);
                break;
        }
        if (intent != null) {
            startActivity(intent);
            LActivityUtil.tranIn(activity);
        }
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/ariana/ArianaMenuActivity.kt
package vn.loitp.app.activity.customviews.ariana

import android.content.Intent
import android.os.Bundle
import android.view.View

import com.core.base.BaseFontActivity
import com.core.utilities.LActivityUtil

import loitp.basemaster.R
import vn.loitp.app.activity.customviews.ariana.iv.ArianaImageViewActivity
import vn.loitp.app.activity.customviews.ariana.tv.ArianaTextViewActivity
import vn.loitp.app.activity.customviews.ariana.vp.ArianaViewPagerActivity

class ArianaMenuActivity : BaseFontActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        findViewById<View>(R.id.bt_ariana_imageview).setOnClickListener(this)
        findViewById<View>(R.id.bt_ariana_textview).setOnClickListener(this)
        findViewById<View>(R.id.bt_ariana_viewpager).setOnClickListener(this)
    }

    override fun setFullScreen(): Boolean {
        return false
    }

    override fun setTag(): String? {
        return javaClass.simpleName
    }

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_menu_ariana
    }

    override fun onClick(v: View) {
        var intent: Intent? = null
        when (v.id) {
            R.id.bt_ariana_imageview -> intent = Intent(activity, ArianaImageViewActivity::class.java)
            R.id.bt_ariana_textview -> intent = Intent(activity, ArianaTextViewActivity::class.java)
            R.id.bt_ariana_viewpager -> intent = Intent(activity, ArianaViewPagerActivity::class.java)
        }
        if (intent != null) {
            startActivity(intent)
            LActivityUtil.tranIn(activity)
        }
    }
}
